lib/sequelize.js

```diff
@@ -1261,6 +1261,7 @@ Sequelize.prototype.or = Sequelize.or;
 Sequelize.prototype.json = Sequelize.json;
 Sequelize.prototype.where = Sequelize.where;
 Sequelize.prototype.validate = Sequelize.prototype.authenticate;
+Sequelize.prototype.Model = Model;
```

Comment:
Please add a typescript typing for this. Observe the above lines:

https://github.com/sequelize/sequelize/blob/50c8a42045d3ea75337d7523ecbc327eb4cffb9d/lib/sequelize.js#L1254-L1262

They all have typings:

https://github.com/sequelize/sequelize/blob/d041e77e275c3e950cedcdb3a625d891ea8b056b/types/lib/sequelize.d.ts#L369-L451

So please add a typing for the newly added `Sequelize.prototype.Model = Model` in the same file. I suggest you do it [after line 451](https://github.com/sequelize/sequelize/blob/d041e77e275c3e950cedcdb3a625d891ea8b056b/types/lib/sequelize.d.ts#L451) to preserve the same positioning.

Reply:
Will do. I just have a question about this 
https://github.com/sequelize/sequelize/blob/50c8a42045d3ea75337d7523ecbc327eb4cffb9d/lib/sequelize.js#L1326

There is static implementation of `Model`. But it's left with no documentation in both files. Maybe it should be documented along with public method or something?

Reply:
I think you can leave it without a JSDoc documentation then. Just add the typing.